docs/getting-started/prerequisites.md

```diff
@@ -10,30 +10,34 @@ To use Atomist, you'll need three things:
     Next at the bottom-right of this page.
 
 [alpha]: https://atomist.typeform.com/to/MseQBs (Atomist Alpha)
-[slack]: https://slack.com/
-[github]: https://github.com
+[slack]: https://slack.com/ (Slack)
+[github]: https://github.com (GitHub)
 
 ## Atomist Alpha
 
-If you have not yet requested access to the Atomist Alpha,
-please fill out the [request form][alpha].  Using the information
-provided in the form, we will reach out to you with further
-instructions for completing your enrollment.  If you have filled out
-the form and have yet to receive the email with further instructions,
-please reach out to us in the <a href="https://join.atomist.com/" alt="join atomist community" target="_blank">Atomist Community Slack</a>
-`#support` channel.
+If you have not yet requested access to the Atomist Alpha, please fill
```

Comment:
yep. Alpha is not a program nor a programme, but a phase of our product rollout. just adds words and doesn't clarify.